portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,26 +19,40 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.Date;
+import com.google.gson.Gson;
+
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        response.setContentType("text/html;");
-        // empty line after the button
-        response.getWriter().println();
-        response.getWriter().println("Hello world!");
-        // The welcome message and show the data
-        response.getWriter().println("Hello, Jiaxi Chen!");
-        // Critical section, using sychronized to make sure mutual exclusion
-        synchronized(this) {
-            count++;
-            response.getWriter().println("You've visited this site " + count +
-                                            " times!");
-        }
+        String json = getInformationInJson();
+        response.setContentType("application/json");
+        response.getWriter().println(json);
     }
 
-    private int count = 0;
+    /**
+     * Creates an ArrayList containing three information, then converts it
+     * json.
+     * @return a String containing the json format of the those information
+     */
+    private static String getInformationInJson() {
+        // creates an ArrayList containing three personal information
```

Comment:
Style guide suggests that comments in the definition should be written in the first person.
```
// Create an ArrayList...
```

Reply:
Fixed.